provision/core/git/sources-ubuntu-bionic.list

```diff
@@ -1,3 +1,3 @@
 # git lfs (large file storage plugin for git)
-deb https://packagecloud.io/github/git-lfs/ubuntu/ bionic main
-deb-src https://packagecloud.io/github/git-lfs/ubuntu/ bionic main
```

Comment:
This change isn't necessary, and wouldn't apply anyway, a `sources-ubuntu-focal.list` is the solution

Reply:
I don't know where that one came from...